noir-projects/noir-contracts/contracts/test_log_contract/src/main.nr

```diff
@@ -40,12 +40,13 @@ pub contract TestLog {
         // We duplicate the emission, but swapping the sender and recipient:
         event0.emit(encode_and_encrypt_event(&mut context, other, context.msg_sender()));
 
-        let event1 = ExampleEvent1 {
-            value2: AztecAddress::from_field(preimages[2]),
-            value3: preimages[3] as u8,
-        };
+        // TODO(benesjan): Emitting this interferes with processing of the logs above. Figure out why.
```

Comment:
This is a bug I did not yet figure out. I did not spend much time on this because I remember that Nico mentioned that it could happen that we would not obtain some notes in some situations and given that we use tagging here as well now I am curious if that could be related.